Makefile

```diff
@@ -128,9 +126,9 @@ test: test-unit test-e2e ## Run all tests.
 .PHONY: test-unit
 test-unit: GOTEST_EXCLUDE := third_party/ test/ hack/ buildenvs/ dist/ docs/
 test-unit: GOTEST_PKGS := $(foreach pkg,$(filter-out $(GOTEST_EXCLUDE),$(wildcard */)),$(pkg)...)
-test-unit: git2go ## Run unit tests.
-	$(GO) run github.com/onsi/ginkgo/v2/ginkgo -v -p -randomize-all --tags static $(GOTEST_PKGS)
 
+test-unit: ## Run unit tests.
+	$(GO) run github.com/onsi/ginkgo/v2/ginkgo -v -p -randomize-all $(GOTEST_PKGS)
```

Comment:
This new line here doesn't look intentional.

Reply:
well you asked me to add it :D

Reply:
I'm talking about the new line above. It isn't there on `main`:

https://github.com/unikraft/kraftkit/blob/2609f3a0b160a555525fc5d76f88fa8cec7afa6e/Makefile#L128-L132